src/openms/source/FORMAT/FileTypes.cpp

```diff
@@ -160,6 +162,27 @@ namespace OpenMS
     return r == FileTypes::Type::UNKNOWN ? fallback : r;
   }
 
+  static FileTypeList typesWithProperties(std::unordered_set<FileTypes::FileProperties> haveFeatures)
+  {
+    std::vector<FileTypes::Type> returnList;
```

Comment:
minor:
- return_list would be correct capitalization
- better name it: compatible because adding return and C++ type to variable names is considered bad naming style